renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": ["config:base"],
+  "semanticCommits": true,
+  "stabilityDays": 3,
+  "prCreation": "not-pending",
+  "labels": ["type: dependencies"],
+  "packageRules": [
+    {
+      "packageNames": ["node"],
+      "enabled": false
+    }
+  ]
+}
```

Comment:
what does this do

Reply:
These are settings for automated dependency updates from Renovate but It need more. Let's delete that for that I guess.